src/garage/tf/policies/categorical_cnn_policy.py

```diff
@@ -1,15 +1,18 @@
-"""CategoricalCNNPolicy with model."""
+"""Categorical CNN Policy.
+
+A policy represented by a Categorical distribution
+which is parameterized by a convolutional neural network (CNN)
+followed a multilayer perceptron (MLP).
+"""
 import akro
+import numpy as np
 import tensorflow as tf
 
-from garage.tf.distributions import Categorical
-from garage.tf.models import CNNModel
-from garage.tf.models import MLPModel
-from garage.tf.models import Sequential
-from garage.tf.policies.policy import StochasticPolicy
+from garage.tf.models import CategoricalCNNModel
+from garage.tf.policies.policy import StochasticPolicy2
 
 
-class CategoricalCNNPolicy(StochasticPolicy):
+class CategoricalCNNPolicy(StochasticPolicy2):
```

Comment:
why still have StochasticPolicy2 if you are getting rid of CategoricalCNNPolicy2, etc?

Reply:
`StochasticPolicy` has `dist_info_sym` which is needed in `GaussianMLPTaskEmbeddingPolicy`.
I am refactoring `GaussianMLPTaskEmbeddingPolicy` rn so `StochasticPolicy2` will be gone later as well.